gulpfile.js

```diff
@@ -129,30 +129,26 @@ gulp.task(
  * express server setup
  */
 gulp.task('serverExpress', function() {
-  // Start the server at the beginning of the task
-  server.run({
-    file: 'server/gulpExpress.js'
-  });
+  var server = gls.new('server/index.js');
+
+  function serverNotify(event) {
+    server.notify.call(server, event);
```

Comment:
call() is needed now, see issue: https://github.com/gimm/gulp-live-server/issues/26